pkg/gadgets/top/file/types/types.go

```diff
@@ -15,25 +15,17 @@
 package types
 
 import (
+	"fmt"
+
 	"github.com/inspektor-gadget/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/inspektor-gadget/inspektor-gadget/pkg/types"
 )
 
-type SortBy int
-
-const (
-	ALL SortBy = iota
-	READS
-	WRITES
-	RBYTES
-	WBYTES
-)
-
 const (
 	AllFilesDefault = false
 )
 
-var SortByDefault = []string{"-reads", "-writes", "-rbytes", "-wbytes"}
+var SortByDefault = []string{"-reads", "-writes", "-r_kb", "-w_kb"}
```

Comment:
`r_kb` and `w_kb` work fine when the output is printed in columns, but what about the user wants to sort the a json output? It seems to be there's something off on this. Perhaps this should keep being rbytes / wbytes, the number on json should always be bytes and we can use `github.com/docker/go-units` in the extractor to print a human readable version of it?

Reply:
Also liking the extractor idea 👍 Implementing it